app/helpers/task_details_helper.rb

```diff
@@ -0,0 +1,32 @@
+module TaskDetailsHelper
+  private
+
+  def row_data_with_link(label, link, value)
+    {:cells => {:label => label, :value => value, :link => link}}
+  end
+
+  def task_details_summary(miq_task, miq_server)
+    rows = [
+      row_data(_('Task ID'), miq_task.id),
+      row_data(_('Task Name'), miq_task.name),
+      row_data(_('State'), _(miq_task.state.titleize)),
+      row_data(_('Status'), _(miq_task.status.titleize)),
+      row_data(_('Message'), _(miq_task.message.titleize)),
```

Comment:
I don't think these on the right side are correct.  For example, I don't think the message contents could ever appear in a catalog, especially titleized, unless I'm not understanding what's in the message.

State and Status might be ok, but all possible values would have to already be in the catalog (and probably defined with N_ somewhere in the code)

Reply:
Should I remove the translation brackets in a follow up PR and just keep them in english? I think you are right about messages not having translations. For state and status not sure if those have translations available either.

Reply:
At least for message, yes.  For the other 2, maybe keep them, but add the hardcoded values into a place the catalog can get them?  Maybe manageiq-providers-workfows?  @agrare Thoughts here?

Reply:
Actually, I'm curious what the other page for the service requests does...we probably should have the same/similar code between the 2.

Reply:
The workflow states table component is the same code for both pages. The only difference between the pages is the details tables.